src/components/Todo/Todo.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Todo.css';
+import PropTypes from 'prop-types';
+
+export class Todo extends React.Component {
+  state = {
+    completed: this.props.completed,
+  }
+
+  handleChange = (event) => {
+    const { name, checked } = event.target;
+
+    this.setState({
+      [name]: checked,
+    });
+  }
```

Comment:
if you add completed/not completed functionality you need to change todo's status in the state too ;)

![image](https://user-images.githubusercontent.com/57598825/97870898-4f338b80-1d1c-11eb-8520-08b8fa1fd8fa.png)